### DISCUSSION TITLE
How to remove sites from General Search

### DISCUSSION THREAD

**Discussion by Axehaft** [searxng/searxng#3091] [Q&A] [answered]
I would like to get results without major news outlets like CNN, ABCNEWS, CBC etc.  I know I can use -site:cnn.com in my search query, but is there a way of automating this or permenantly adding these modifiers?

**Comment by virtadpt** [comment]
The easiest way is probably in post-processing, if you're using the API.

**Reply by Axehaft** [reply]
I'm not sure what that means or how to do that.  I'm running my own instance of Searxng with Yunohost.  I was hoping I could set something there.

**Comment by return42** [comment, marked as answer]
> I would like to get results without major news outlets like CNN, ABCNEWS, CBC etc. I know I can use -site:cnn.com in my search query,

This `-site:cnn.com` will only work for a few engines

> but is there a way of automating this or permenantly adding these modifiers?

Yes, "hostname_replace" plugin

- https://github.com/searxng/searxng/discussions/970

https://github.com/searxng/searxng/blob/3535377c9aa777d95888a853d813e94fbbae5489/searx/settings.yml#L227-L237
